CODEOWNERS

```diff
@@ -113,6 +113,7 @@
 /drivers/counter/                         @nordic-krch
 /drivers/display/                         @vanwinkeljan
 /drivers/dma/*sam0*                       @Sizurka
+/drivers/espi/                            @albertofloyd @franciscomunoz
```

Comment:
remember we are only owners of mchp_espi_xec. We already have an expression for all the drivers   /drivers/*/*xec* . So this line should be removed